core/deployment/src/main/java/io/quarkus/deployment/pkg/PackageConfig.java

```diff
@@ -248,10 +248,10 @@ public static BuiltInType fromString(String value) {
     public boolean includeDependencyList;
 
     /**
-     * Quiltflower Decompiler configuration
+     * Vineflower Decompiler configuration
      */
     @ConfigItem
-    public QuiltFlowerConfig quiltflower;
+    public VineFlowerConfig vineflower;
```

Comment:
I think it's somewhat meant to be futureproof in the sense you could have support for multiple decompilers for example the config is `quarkus.package.vineflower.enabled=true` maybe there could be support for another one this way? `quarkus.package.my-other-decompiler.enabled=true`. I may be wrong though!

a bit what I could gather from the code in https://github.com/quarkusio/quarkus/pull/25729 where both fernflower and quiltflower could coexist

Reply:
The reason I ask is that I never know the config to enable the compiler without looking at the docs. `quarkus.package.decompiler.enabled=true` is cleaner (to me at least) 😄

Reply:
> a bit what I could gather from the code in #25729 where both fernflower and quiltflower could coexist

Right, and I believe this is the reason why @geoand made it that way. I'm afraid that removing the config option would make existing users a bit annoyed, so perhaps deprecating (for removal) `quarkus.package.quiltflower` in favor of `quarkus.package.vineflower` may be better in this case

Reply:
But I am still leaning towards having `quarkus.package.decompiler.enabled`, given I doubt anybody cares about the decompiler used in practice :)

Reply:
So is there a consensus here? :) 

In my opinion, having both config,  with the deprecated one is more or less useful as there isn't extreme adaptation to be made. On quarkus update, when looking at the migration it would be quite clear, vs deprecating, it, then doing it in some other release, not like if a behaviour changed underneath. As per `quarkus.package.decompiler.enabled`, I do like it, but definitely not my call! I can adapt to what you guys decide.

Reply:
I'd like to keep the decompiler name for the time being.